src/routes/categories/components/CategoryList.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import CategoryModel from 'modules/endpoint/models/CategoryModel'
+import RemoteContent from 'modules/common/components/RemoteContent'
+
+import ContentList from './ContentList'
+
+import style from './CategoryList.css'
+
+/**
+ * Displays a ContentList which is a list of categories, a caption and a thumbnail
+ */
+class CategoryList extends React.Component {
+  static propTypes = {
+    parentCategory: PropTypes.instanceOf(CategoryModel).isRequired,
+    categories: PropTypes.arrayOf(PropTypes.instanceOf(CategoryModel)).isRequired
```

Comment:
This will also be simple if we have a Categorie_s_Model

Reply:
This is now a CategoriesContainer, right?

Reply:
no, we don't need mapping to a url here anymore, because this just displays a list of categoryModels (the children of the current category), so a CategoriesContainer does not make sense here